tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/ServiceRecipients Put.feature

```diff
@@ -124,3 +124,13 @@ Scenario: 10. Service Failure
         | OdsCode | Name         |
         | Ods2    | Another Name |
     Then a response with status code 500 is returned
+
+@7412
+Scenario: 11. the user selects service recipients and the order marks the service recipient section as viewed
```

Comment:
Might be a good idea to move this further up, to follow the pattern of authorisation tasks then the service failure. But up to you

Reply:
I thought about it but then I really didn't want to have to change the numbers :-)

Reply:
Of course you didnt... 😄